### DISCUSSION TITLE
Issue with Mapsets with a custom Skin and no custom Reverse Slider

### DISCUSSION THREAD

**Discussion by Kurboh** [ppy/osu#20637] [Ideas] [answered]
Reverse Arrows didn't appear for both the "triangles" and "argon" Skin while playing the Mapset with Custom Skin turned on, both in game and in the editor.
This Mapset has a custom Hitcircle and no custom Reverse Arrow, the only hint for there being a Reverse there is a suttle shake when the Slider is done forming, but I feel like that isn't enough for players to react to.
![osu_2022-10-08_16-43-35](https://user-images.githubusercontent.com/111684932/194713288-662b4958-ce33-4576-b0e9-96f3beddb840.jpg)
Editor preview with "argon (& triangles)" skin
![osu_2022-10-08_16-45-26](https://user-images.githubusercontent.com/111684932/194713336-4231fe3d-04d0-4699-be32-489990aa4936.jpg)
Editor preview with "classic" skin

**Comment by peppy** [comment, marked as answer]
What beatmap
Where logs
help compute

**Reply by smoogipoo** [reply]
I recognised the beatmap from the background (pretty nostalgic) :smile: https://github.com/ppy/osu/issues/20710

https://osu.ppy.sh/beatmapsets/33842#osu/110175
